src/modules/room/type/type.schema.ts

```diff
@@ -1,30 +1,33 @@
 import { z } from "zod";
 import { buildJsonSchemas } from "fastify-zod";
+import { roomSchema } from "../room.schema";
 
 const roomTypeCore = {
-    name: z.string({
-        required_error: "Room type name is required",
-        invalid_type_error: "Name must be a string",
-    }).min(1),
+    name: z
+        .string({
+            required_error: "Room type name is required",
+            invalid_type_error: "Name must be a string",
+        })
+        .min(1),
     description: z.string().nullable().optional(),
     size: z.string().nullable().optional(),
     price: z.number(),
     hotelId: z.number(),
 };
```

Comment:
We're missing people_amount, please add.. :)
![image](https://user-images.githubusercontent.com/27145523/221544971-64bdc703-6886-41a3-9b00-59df749ae57c.png)